addons/hr_holidays/static/tests/leave_stats_tests.js

```diff
@@ -135,7 +135,7 @@ QUnit.test("leave stats renders correctly", async (assert) => {
     assert.containsOnce($leaveTypeBody, "span:contains(6)");
     assert.containsN($leavesDepartmentBody, "span:contains(Richard)", 2);
     assert.containsOnce($leavesDepartmentBody, "span:contains(Jesus)");
-    assert.containsOnce($leavesDepartmentBody, "div.o_horizontal_separator:contains(R&D)");
+    assert.containsOnce($leavesDepartmentBody, "h3.o_hr_leave_subtitle.border-bottom:contains(R&D)");
```

Comment:
commit 771dc529383f66446aa307baada638a0c9ce192c
This selector is too complex and fragile as it would break as soon as we replace an utility class, you could make use of the `o_leave_stats_department` ID on the parent element to identify your targeted h3.